src/test/java/org/springframework/cloud/dataflow/acceptance/test/AbstractStreamTests.java

```diff
@@ -39,30 +39,33 @@
 import org.springframework.cloud.dataflow.rest.client.StreamOperations;
 import org.springframework.cloud.dataflow.rest.resource.StreamDefinitionResource;
 import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.util.Assert;
 import org.springframework.util.StringUtils;
 import org.springframework.web.client.RestTemplate;
 
 /**
- * Abstract base class that is used by stream acceptance tests.  This class
- * contains commonly used utility methods for acceptance tests as well as the
- * ability to dump logs of apps when a stream acceptance test fails.
+ * Abstract base class that is used by stream acceptance tests. This class contains
+ * commonly used utility methods for acceptance tests as well as the ability to dump logs
+ * of apps when a stream acceptance test fails.
+ *
  * @author Glenn Renfro
+ * @author Vinicius Carvalho
  */
 @RunWith(SpringRunner.class)
 @EnableConfigurationProperties(TestConfigurationProperties.class)
 public abstract class AbstractStreamTests implements InitializingBean {
```

Comment:
I'll add checkstyle and the eclipse formatting rules to the code base to avoid the diff noise wrt formatting.